File: c++/2699 - Measuring December Rain.cpp
#include <cmath>
#include <cstdio>
#include <string>
#include <iostream>

int main()
{
	int i = 0, input;
	float acum = 0.f;
	for (; i < 30; ++i)
	{
		scanf("%d", &input);
		acum += input;
	}
	float average = acum / 30.0f;
	acum += average;
	acum = ceil(acum * 1000) / 1000;
	printf("%.3f\n", acum);
}

File: Java/2699 - Measuring December Rain.java
import java.util.Scanner;


public class Main {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int i = 0, input;
		double acum =0;
		for(;i<30;++i){
			input = in.nextInt();
			acum += input;
		}
		double average = acum / 30.0;
		acum += average;
		System.out.printf("%.3f\n",acum);
	}
}
